src/main/java/com/sonyericsson/jenkins/plugins/bfa/PluginImpl.java

```diff
@@ -155,6 +156,7 @@
     private String fallbackCategoriesAsString;
     private transient List<String> fallbackCategories;
 
+    private Boolean exportCausesToJCasCEnabled;
```

Comment:
This can safely be `boolean` as that would default to `false` when old config is migrated. Consuming code can be simplified.